python/private/toolchains_repo.bzl

```diff
@@ -301,20 +297,20 @@ this repo causes an eager fetch of the toolchain for the host platform.
 )
 
 def _multi_toolchain_aliases_impl(rctx):
-    rules_python = rctx.attr._rules_python_workspace.workspace_name
+    rules_python = rctx.attr._rules_python_workspace.repo_name
 
     for python_version, repository_name in rctx.attr.python_versions.items():
         file = "{}/defs.bzl".format(python_version)
         rctx.file(file, content = """\
 # Generated by python/private/toolchains_repo.bzl
 
-load("{rules_python}//python:pip.bzl", _compile_pip_requirements = "compile_pip_requirements")
-load("{rules_python}//python/private:deprecation.bzl", "with_deprecation")
-load("{rules_python}//python/private:text_util.bzl", "render")
-load("{rules_python}//python:py_binary.bzl", _py_binary = "py_binary")
-load("{rules_python}//python:py_test.bzl", _py_test = "py_test")
+load("@@{rules_python}//python:pip.bzl", _compile_pip_requirements = "compile_pip_requirements")
```

Comment:
Does it always make sense to load from the global root module context?

Reply:
You mean that there are still some other places in this file that use `@rules_python` instead of `@@{rules_python}`? Well, I thought that looked a bit funky, but wasn't sure whether that was accidental/intentional. Happy to change the code either way.